hack/kind/setup.sh

```diff
@@ -77,11 +77,21 @@ setup_cluster() {
   kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
 }
 
+install_kubectl() {
+    if ! command -v kubectl &> /dev/null
+then
+    info "kubectl not found, attempting to install"
```

Comment:
```suggestion
    if ! command -v kubectl >& /dev/null; then  # 2 corrections >& and nit:  ; then to follow the convention
      info "kubectl not found, attempting to install"
```

Reply:
According to `man bash` `&>` is the preferred version of the otherwise equivalent versions.

Reply:
wow! didn't know this was the preferred option.  Force of habit :)